src/components/transcript/transcript.tsx

```diff
@@ -1,14 +1,14 @@
 import {h, Component} from 'preact';
-import {OnClickEvent, OnClick} from '@playkit-js/common/dist/hoc/a11y-wrapper';
 import {debounce} from '../../utils';
 import * as styles from './transcript.scss';
 import {Spinner} from '../spinner';
 import {Search} from '../search';
 import {CaptionList} from '../caption-list';
 import {HighlightedMap, CuePointData} from '../../types';
-import {CloseButton} from '../close-button';
 import {ErrorIcon} from './error-icon';
 import {AutoscrollButton} from '../autoscroll-button';
+import {TranscriptMenu} from '../transcript-menu';
+import {Button, OnClickEvent, OnClick, ButtonType, ButtonSize} from '@playkit-js/common';
```

Comment:
don't use import from '@playkit-js/common' because it contains all component and utils from common repo. Instead import only components and methods that you need:

import {OnClickEvent, OnClick} from '@playkit-js/common/dist/hoc/a11y-wrapper'
import {Button, ButtonType, ButtonSize} from '@playkit-js/common/dist/components/button'